### PR TITLE
Add Iris / dask / distributed post

### PR DESCRIPTION
**Pull request by dkillick**
At long last, a blog post about the daskified Iris investigation I performed while visiting the lab.

~~Still needs images...~~

### CODE REVIEW

File: _posts/2017/iris_dask_distributed_on_spice.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
++---
++title: Iris, dask, distributed and SPICE
++date: 2017-12-14 00:00:00 Z
++categories:
++- dask
++layout: post
++summary: Massively parallel processing of large datasets using Iris.
++author:
```
Comment (jacobtomlinson) [inline _posts/2017/iris_dask_distributed_on_spice.md]: Add your name here 😄

File: _posts/2017/iris_dask_distributed_on_spice.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
++project:
```
Comment (jacobtomlinson) [inline _posts/2017/iris_dask_distributed_on_spice.md]: Possibly Jade? Otherwise leave blank.

File: _posts/2017/iris_dask_distributed_on_spice.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
++thumbnail:
```
Comment (jacobtomlinson) [inline _posts/2017/iris_dask_distributed_on_spice.md]: We should find something for the thumbnail and header. [Pexels](https://www.pexels.com/) is a good source of public domain images that you can use.

File: _posts/2017/iris_dask_distributed_on_spice.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+distributed on SPICE to the number of worker processes we can set up.
+
+This has no short-term fix (other than allocating more CPU cores!), but in the
+longer term we can look to partner with the authors of the problem dependency
+libraries to improve the functionality of these dependency libraries. It may even
+be that all that is needed is a rebuild of the libraries that are not thread-safe,
+which would bring a whole lot of benefit for not too much work!
+
+
+### In Conclusion
```
Comment (jacobtomlinson) [inline _posts/2017/iris_dask_distributed_on_spice.md]: Lowercase `c`?

File: _posts/2017/iris_dask_distributed_on_spice.md (hunk 2 above)
Comment (dkillick) [reply _posts/2017/iris_dask_distributed_on_spice.md]: It's not _quite_ Jade, so I might leave this one blank.

Comment (jacobtomlinson) [reply _posts/2017/iris_dask_distributed_on_spice.md]: That's fair

**Comment by tam203** [comment]
👍 Agree pictures would be nice. Could be a picture of slurm maybe... or a dask job on jupyterhub and just pretend it's from spice...

**Comment by dkillick** [comment]
Update: now with images! 🖼

**Comment by dkillick** [comment]
🎉

**Comment by mrocklin** [comment]
Hey, if you all tweet these blogposts when you write them and cc `@dask_dev` and we'll retweet them.  

It would be good to get more community exposure on this sort of work.

**Pull request merged**